pyvista/utilities/geometric_objects.py

```diff
@@ -1218,6 +1219,77 @@ def Circle(radius=0.5, resolution=100):
     return pyvista.wrap(pyvista.PolyData(points, cells))
 
 
+def Superquadric(center=(0., 0., 0.), scale=(1., 1., 1.), size=0.5,
+                 theta_roundness=1., phi_roundness=1.,
+                 theta_resolution=16, phi_resolution=16,
+                 toroidal=False, thickness=1/3):
+    """Create a superquadric.
+
+    Parameters
+    ----------
+    center : iterable, optional
+        Center of the superquadric in ``[x, y, z]``.
+
+    scale : iterable, optional
+        Scale factors of the superquadric in ``[x, y, z]``.
+
+    size : float, optional
+        Superquadric isotropic size.
+
+    theta_roundness : float, optional
+        Superquadric east/west roundness.
+        Values range from 0 (rectangular) to 1 (circular) to higher orders.
+
+    phi_roundness : float, optional
+        Superquadric north/south roundness.
+        Values range from 0 (rectangular) to 1 (circular) to higher orders.
+
+    theta_resolution : int, optional
+        Number of points in the longitude direction.
+        Values are rounded to nearest multiple of 4.
+
+    phi_resolution : int, optional
+        Number of points in the latitude direction.
+        Values are rounded to nearest multiple of 8.
+
+    toroidal : bool, optional
+        Whether or not the superquadric is toroidal (1) or ellipsoidal (0).
+
+    thickness : float, optional
+        Superquadric ring thickness.
+        Only applies if toroidal is set to 1.
+
+    Returns
+    -------
+    pyvista.PolyData
+        Superquadric mesh.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> superquadric = pyvista.Superquadric()
+    >>> superquadric.plot(show_edges=True)
+
+    See Also
+    --------
+    pyvista.ParametricSuperEllipse : Parametric superquadric if toroidal is 0.
+    pyvista.ParametricSuperToroid : Parametric superquadric if toroidal is 1.
```

Comment:
Numpydoc says
> WARNING: [numpydoc] Validation warnings while processing docstring for 'pyvista.ParametricSuperEllipsoid':
> GL07: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
(this makes CI fail)

Reply:
Fixed now - before I push and trigger another failed CI run: How do I test numpydoc locally? I'm not sure if `pyvista.ParametricSuperEllipse` etc are the correct links, or if that should be a local/relative path.

Reply:
> How do I test numpydoc locally? I'm not sure if pyvista.ParametricSuperEllipse etc are the correct links, or if that should be a local/relative path.

Unfortunately, the only way I know how is to build the entire documentation locally.  I did a quick survery of other "See also" sections and they are all using the `pyvista.X` form.  This also makes it clear to the user that it is available in the `pyvista` namespace.

Reply:
@janniklasrose for the record, you first have to have the docs requirements installed (`pip install -r requirements_doc.txt`, preferably in a dedicated virtual environment because this installs a lot of cruft you otherwise don't need), then `make -C doc clean html` from the `pyvista` directory (or jut `make clean html` from the `pyvista/doc` directory). It will take quite some time, and it might not build on VTK 9.1.0.